app/models/user.rb

```diff
@@ -11,12 +11,28 @@ class User < RedisStore
   attribute :last_name
   attribute :zip
 
+  # vaafi attributes
+  attribute :edipi
+  attribute :issue_instant
+  attribute :participant_id
+  attribute :ssn
+
   # Add additional MVI attributes
   alias redis_key uuid
 
   validates :uuid, presence: true
   validates :email, presence: true
 
+  def self.sample_claimant
+    sample_claimant = User.new
+    sample_claimant.first_name = 'Jane'
+    sample_claimant.last_name = 'Doe'
+    sample_claimant.issue_instant = '2015-04-17T14:52:48Z'
+    sample_claimant.edipi = '1105051936'
+    sample_claimant.participant_id = '123456789'
+    return sample_claimant
```

Comment:
I will try that. I figured that the way I had this written was not the _Ruby Way_ because it was too verbose, but I couldn't get a `do` block to assign to the attributes, and nothing turned up in Google for "Ruby Javascript with statement equivalent".